docs/source/_json/users.json

```diff
@@ -0,0 +1,26 @@
+GET /plone/@users
+Accept: application/json
+
+HTTP 200 OK
+content-type: application/json
+
+[
```

Comment:
@lukasgraf of course! I started with a proof-of-concept implementation. Sorry, should have marked this PR as WIP.